src/config/index.js

```diff
@@ -3,7 +3,7 @@ import ENV from '../env';
 const config = {
     API_URL: 'https://api.spotify.com/v1',
     SPOTIFY_AUTHORIZE_URL: "https://accounts.spotify.com/authorize",
-    SPOTIFY_AUTH_SCOPES: "user-read-recently-played user-top-read playlist-modify-public",
+    SPOTIFY_AUTH_SCOPES: "user-read-recently-played user-top-read playlist-read-private playlist-modify-public",
```

Comment:
We might get away with the scope 'playlist-read-public' instead of 'playlist-read-private' but we can make that decision later